abci/example/kvstore/persistent_kvstore.go

```diff
@@ -170,6 +170,25 @@ func (app *PersistentKVStoreApplication) ApplySnapshotChunk(
 	return types.ResponseApplySnapshotChunk{Result: types.ResponseApplySnapshotChunk_ABORT}
 }
 
+func (app *PersistentKVStoreApplication) PrepareProposal(
+	req types.RequestPrepareProposal) types.ResponsePrepareProposal {
+	// if len(req.BlockData.Txs) >= 1 {
+	// 	req.BlockData.Txs[0] = []byte("modified tx")
+	// }
```

Comment:
?

Reply:
I removed this comment, it was added in the original PR for a test I think, but then removed in later refactors. I originally commented it out, as it was causing tests to fail.